clang/include/clang/Driver/Options.td

```diff
@@ -4099,6 +4099,10 @@ def fsycl_host_compiler_options_EQ : Joined<["-"], "fsycl-host-compiler-options=
   Visibility<[ClangOption, CLOption, DXCOption]>, HelpText<"When performing the host compilation with "
   "-fsycl-host-compiler specified, use the given options during that compile. "
   "Options are expected to be a quoted list of space separated options.">;
+def fsycl_llc_options_EQ : Joined<["-"], "fsycl-llc-options=">,
```

Comment:
If this is just temporary - add `Flags<[HelpHidden]>` to reduce potential exposure